src/LuaGame.cpp

```diff
@@ -63,8 +63,13 @@ static int l_game_start_game(lua_State *l)
 	}
 
 	SystemPath *path = LuaObject<SystemPath>::CheckFromLua(1);
-	const double start_time = luaL_optnumber(l, 2, 0.0);
```

Comment:
A short API documentation compatible paragraph might be nice, or is this only for internal use? Any chance someone else might want to use this?

Reply:
It's an old function that is documented just above. It's always been in there but I don't think anyone uses it.